puzzlejs/puzzle.js

```diff
@@ -1512,7 +1576,15 @@ function PuzzleGrid(puzzleEntry, options, container, puzzleId) {
     var solution = puzzleEntry.getOptionArray(this.options, "data-text-solution", "|");
     var edges = puzzleEntry.getOptionArray(this.options, "data-edges", "|");
     var paths = puzzleEntry.getOptionArray(this.options, "data-paths", "|");
-    var spokes = puzzleEntry.getOptionArray(this.options, "data-spokes", "|");
+    var maxSpokeLevels = parseInt(this.options["data-spoke-level"]) + 1;
+    var spokes = new Array(maxSpokeLevels);
+    for (var l = 1; l < maxSpokeLevels; l++) {
+        var spoke;
+        if (l > 1) { spoke = puzzleEntry.getOptionArray(this.options, "data-spokes-" + l.toString(), "|"); }
+        else { spoke = puzzleEntry.getOptionArray(this.options, "data-spokes", "|"); }
+        if (!spoke) { spoke = 0; }
```

Comment:
Why 0 and not, say, null?

Reply:
copy paste error!  Actually, since it's being used as a string, I'll go with an empty string.